app/parsers/bulkrax/csv_parser.rb

```diff
@@ -391,12 +396,12 @@ def sort_entries(entries)
       # always export models in the same order: work, collection, file set
       entries.sort_by do |entry|
         case entry.type
-        when 'Bulkrax::CsvEntry'
-          '0'
         when 'Bulkrax::CsvCollectionEntry'
           '1'
         when 'Bulkrax::CsvFileSetEntry'
           '2'
+        else
```

Comment:
in apps that have custom work entry types, they should all be counted as a "Bulkrax::CsvEntry". this is in line with what we're doing in `#create_new_entries` too.